From b8ab14d5a149304ccaf6ed88c5338fdcb279cff8 Mon Sep 17 00:00:00 2001
From: Avinash Reddy Palleti <avinash.reddy.palleti@intel.com>
Date: Thu, 9 Nov 2017 10:59:07 +0530
Subject: [PATCH] Update postinst script to change default grub

---
 scripts/package/builddeb | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index 6c3b038..98e98ae 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -201,6 +201,13 @@ else
 fi
 for script in postinst postrm preinst prerm ; do
 	mkdir -p "$tmpdir$debhookdir/$script.d"
+if [ $script == "postinst" ]; then
+	grub_string="sed -i 's/GRUB_DEFAULT=0/GRUB_DEFAULT=\"Advanced options for Ubuntu>Ubuntu, with Linux 4.4.76-aero-1.3\"/' /etc/default/grub 
+sed -i 's/GRUB_HIDDEN_TIMEOUT=0/#GRUB_HIDDEN_TIMEOUT=0/' /etc/default/grub
+update-grub" 
+else
+	grub_string=""
+fi
 	cat <<EOF > "$tmpdir/DEBIAN/$script"
 #!/bin/sh
 
@@ -213,6 +220,7 @@ export DEB_MAINT_PARAMS="\$*"
 export INITRD=$want_initrd
 
 test -d $debhookdir/$script.d && run-parts --arg="$version" --arg="/$installed_image_path" $debhookdir/$script.d
+$grub_string
 exit 0
 EOF
 	chmod 755 "$tmpdir/DEBIAN/$script"
-- 
2.7.4

